app/frontend/src/App.tsx

```diff
@@ -1,4 +1,4 @@
-import { Container, CssBaseline, ThemeProvider } from "@material-ui/core";
+import { Container, CssBaseline, Hidden, ThemeProvider, makeStyles } from "@material-ui/core";
```

Comment:
Please run `yarn format` to run prettier which will split this up into multiple lines now it's longer.

If you're using vscode, you can set it to run it automatically on save (that's what I do) by putting this in your preferences.json:

```
    "editor.formatOnSave": true,
    "[typescriptreact]": {
        "editor.defaultFormatter": "esbenp.prettier-vscode"
    }
```

Reply:
(This is after installing the prettier extension)

Reply:
My bad Lucas! I had installed prettier and wrongly assumed it was already running on save.